app/controllers/concerns/lock_ingestible_concern.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Logic to lock Ingestible object, to be included in related controllers
+module LockIngestibleConcern
```

Comment:
I've moved this method to separate concern because I expect some other controllers to work with ingestibles in future